css/style.css

```diff
@@ -0,0 +1,369 @@
+@import url('/css/fonts.css');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-image: url(/img/Background.png);
+  background-size: cover;
+  font-family: 'Roboto Bold', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.container {
+  max-width: 70%;
+  margin: auto;
+}
+
+.game {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  align-items: center;
+}
+
+.menu {
+  width: 100%;
+  position: relative;
+}
+
+.level-choice__header {
+  margin-bottom: 16px;
+  font-size: 54px;
+  color: #FFFFFF;
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)